knowage-vue/src/modules/documentExecution/olap/filterPanel/OlapFilterCard.vue

```diff
@@ -3,7 +3,13 @@
         <div :id="'filter-' + filter.name" :ref="'filter-' + filter.name" :style="panelDescriptor.style.filterCard" draggable="true" @dragstart="onDragStart($event, filter, 'filter-' + filter.name)" @dragend="removeDragClass('filter-' + filter.name)">
             <Button v-if="filter.hierarchies.length > 1" icon="fas fa-sitemap" class="p-button-text p-button-rounded p-button-plain" @click="$emit('showMultiHierarchy', filter)" />
             <span class="p-ml-1"> {{ filter.caption }} </span>
-            <Button icon="fas fa-filter" :class="{ 'olap-active-filter-icon': filterIsActive(filter) }" class="p-button-text p-button-rounded p-button-plain p-ml-auto" @click="openFilterDialog(filter)" />
+            <Button
+                icon="fas fa-filter"
+                :class="{ 'olap-active-filter-icon': filterIsActive(filter) }"
+                class="p-button-text p-button-rounded p-button-plain p-ml-auto"
+                v-tooltip="{ value: getSlicersTooltip(filter), disabled: !filter || !filter.hierarchies[0].slicers || filter.hierarchies[0].slicers.length === 0 }"
+                @click="openFilterDialog(filter)"
+            />
             <!-- TODO Change Request for next sprint: Tooltip for selected filters when hovering on icon and knowage magenta button color if filter is selected -->
```

Comment:
I see that the toolpit has been already added. Do I need to remove the TODO comment?

Reply:
I think Darko made that comment. @dbulatovicx32

Reply:
You can remove it yes.

Reply:
You can remove it.